pack_and_terraform.bash

```diff
@@ -149,6 +149,10 @@ if echo "$RUNTF" | grep -iq "^y"; then
 	HMAC=`cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1 | base64`
 	echo "Your HMAC encryption key is: $HMAC"
 
+
+	GDCAPI_SECRET=`cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 50 | head -n 1`
```

Comment:
I think this should be:
```sh
base64 /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 50 | head -n 1
```
(at least on my machine this way works but `cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 50 | head -n 1` does not)